fbw-a32nx/src/systems/fmgc/src/flightplanning/rpc/FlightPlanRpcClient.ts

```diff
@@ -74,19 +124,54 @@ export class FlightPlanRpcClient<P extends FlightPlanPerformanceData> implements
     funcName: T,
     ...args: Parameters<FunctionsOnlyAndUnwrapPromises<FlightPlanInterface<P>>[T]>
   ): Promise<ReturnType<FunctionsOnlyAndUnwrapPromises<FlightPlanInterface<P>>[T]>> {
-    const id = v4();
+    const batchState = { batch: null, state: false };
```

Comment:
What does `state` mean here? It's not clear to me what this boolean signifies. Maybe we can choose a more descriptive name?